README.md

```diff
@@ -36,6 +36,19 @@ You probably want to include the default stylesheet, too. It displays error mess
 
 If you're not a fan of the default styling, you can drop in your own stylesheet. The only new element to worry about is `vjs-errors-dialog` which is the container for the error messages.
 
+When using via **NPM/Browserify**
+```shell
+npm install videojs-errors
+```
+
+You need to add videojs-errors as plugin to the videojs module:
+```js
+import videojs from 'video.js';
+import videojsErrors from 'videojs-errors';
+
+videojs.plugin('errors', videojsErrors);
```

Comment:
Thanks @misteroneill for the clarification. It seems I'm not getting the same videojs object, with the attached videojsErrors plugin, after importing the module. Let me run through some tests and update/close this PR. Thanks again.